src/app/imgui_remote.h

```diff
@@ -15,485 +15,551 @@
-						Input.MousePos.y = (float)y;
-                        Input.MouseButtons = mouse_left | (mouse_right << 1);
+						Input.MousePos.x = (float) x;
+						Input.MousePos.y = (float) y;
+						Input.MouseButtons = mouse_left | ( mouse_right << 1 );
 					}
 				}
-				else if (strstr((char *)data, "ImMousePress"))
+				else if ( strstr( (char *) data, "ImMousePress" ) )
 				{
 					int l, r;
-					if (sscanf((char *)data, "ImMousePress=%d,%d", &l, &r) == 2)
+					if ( sscanf( (char *) data, "ImMousePress=%d,%d", &l, &r ) == 2 )
 					{
 						FrameReceived = Frame;
-						Input.MouseButtons = l | (r<<1);
+						Input.MouseButtons = l | ( r << 1 );
 					}
 				}
-				else if (strstr((char *)data, "ImMouseWheelDelta"))
+				else if ( strstr( (char *) data, "ImMouseWheelDelta" ) )
 				{
 					float mouseWheelDelta;
-					if (sscanf((char *)data, "ImMouseWheelDelta=%f", &mouseWheelDelta) == 1)
+					if ( sscanf( (char *) data, "ImMouseWheelDelta=%f", &mouseWheelDelta ) == 1 )
 					{
 						FrameReceived = Frame;
 						Input.MouseWheelDelta = mouseWheelDelta *0.01f;
 					}
 				}
-				else if (strstr((char *)data, "ImKeyDown"))
+				else if ( strstr( (char *) data, "ImKeyDown" ) )
 				{
 					int key, shift, ctrl;
-					if (sscanf((char *)data, "ImKeyDown=%d,%d,%d", &key, &shift, &ctrl) == 3)
+					if ( sscanf( (char *) data, "ImKeyDown=%d,%d,%d", &key, &shift, &ctrl ) == 3 )
 					{
 						//update key states
 						FrameReceived = Frame;
 						Input.KeyShift = shift > 0;
 						Input.KeyCtrl = ctrl > 0;
-                        mapRemoteKey(&key, Input.KeyCtrl);
-						Input.KeysDown[key] = true;
+						mapRemoteKey( &key, Input.KeyCtrl );
+						Input.KeysDown[ key ] = true;
 					}
 				}
-				else if (strstr((char *)data, "ImKeyUp"))
+				else if ( strstr( (char *) data, "ImKeyUp" ) )
 				{
 					int key;
-					if (sscanf((char *)data, "ImKeyUp=%d", &key) == 1)
+					if ( sscanf( (char *) data, "ImKeyUp=%d", &key ) == 1 )
 					{
 						//update key states
 						FrameReceived = Frame;
-						Input.KeysDown[key] = false;
+						Input.KeysDown[ key ] = false;
 						Input.KeyShift = false;
 						Input.KeyCtrl = false;
 					}
 				}
-				else if (strstr((char *)data, "ImKeyPress"))
+				else if ( strstr( (char *) data, "ImKeyPress" ) )
 				{
-                    unsigned int key;
-					if (sscanf((char *)data, "ImKeyPress=%d", &key) == 1)
-						ImGui::GetIO().AddInputCharacter(ImWchar(key));
+					unsigned int key;
+					if ( sscanf( (char *) data, "ImKeyPress=%d", &key ) == 1 )
+						ImGui::GetIO().AddInputCharacter( ImWchar( key ) );
 				}
-				else if (strstr((char *)data, "ImClipboard="))
+				else if ( strstr( (char *) data, "ImClipboard=" ) )
 				{
-					char *clipboard = &((char *)data)[strlen("ImClipboard=")];
-					ImGui::GetIO().SetClipboardTextFn(nullptr, clipboard);
+					char *clipboard = &( (char *) data )[ strlen( "ImClipboard=" ) ];
+					ImGui::GetIO().SetClipboardTextFn( clipboard );
```

Comment:
This seems to have changed in later versions.

Reply:
Sorry for the delay, looking good! Thanks for your contribution